apps/web/vibes/soul/sections/account-settings-section/update-account-form.tsx

```diff
@@ -67,17 +150,22 @@ export function UpdateAccountForm({
         key={fields.email.id}
         label={emailLabel}
       />
-      <SubmitButton>{submitLabel}</SubmitButton>
+      <Input
+        {...getInputProps(fields.company, { type: 'text' })}
+        errors={fields.company.errors}
+        key={fields.company.id}
+        label={companyLabel}
+      />
```

Comment:
I initially planned and changed this component to account for that, but realized that BigCommerce doesn't have custom account fields, only custom address and register customer fields. Because it wasn't dynamic and there are only a few fields here, I opted to keep it simple but if we see a need in the future we can probably shift it back over to dynamic fields.

Reply:
Yup, not something for us to worry about right now. VIBES has only 1 user, and it's Catalyst. If we have a need for different fields we can always use `DynamicForm` and have the consumer of VIBES pass in the fields, even if they're statically defined and not coming from an API